assets/js/base/components/checkbox-list/index.js

```diff
@@ -48,17 +48,21 @@ const CheckboxList = ( {
 						} }
 						aria-expanded={ false }
 						aria-label={ sprintf(
-							__(
+							_n(
+								'Show %s more option',
 								'Show %s more options',
+								optionCount - limit,
 								'woo-gutenberg-products-block'
 							),
 							optionCount - limit
 						) }
 					>
 						{ // translators: %s number of options to reveal.
 						sprintf(
-							__(
+							_n(
```

Comment:
I know this wasn't added in this pull but the translators comment is in the wrong place here.  It should be in the line above the `_n`